includes/weather-functions.php

```diff
@@ -37,9 +37,21 @@ function get_weather( $cache_key ) {
 		}
 
 		if ( $json_url ) {
-			$json = array();
-			$args = array(
-				'timeout' => get_option( 'weather_service_timeout' )
+			// Setup curl request and execute. Log errors if necessary.
+
+			$dt = urlencode( microtime() );
+
+			if ( strpos( $json_url, '?' ) > -1 ) {
+				$json_url .= "&request_time=$dt";
+			} else {
+				$json_url .= "?request_time=$dt";
+			}
+
+			$ch = curl_init();
+			$options = array(
```

Comment:
`$options` is unused here/should be renamed `args`.  Additionally, curl arg names need to be modified to fit expected arg names for `wp_remote_get()`: https://developer.wordpress.org/reference/classes/WP_Http/request/